app/controllers/integrations/buildkite_controller.rb

```diff
@@ -28,4 +28,9 @@ def build_param
   def message
     build_param[:message]
   end
+
+  def release_params
+    extra_params = Samson::Hooks.fire(:release_params, project, build_param)
```

Comment:
I see, thanks :)  I've changed the hook name to be buildkite_release_params in the latest commit, please take a look.